fastlane/lib/fastlane/helper/crashlytics_helper.rb

```diff
@@ -72,8 +72,12 @@ def download_android_tools
           begin
             UI.important("Downloading Crashlytics Support Library - this might take a minute...")
 
-            result = Net::HTTP.get(URI(url))
-            File.write(zip_path, result)
+            uri = URI(url)
```

Comment:
It would be nice to mention (with a link to the [ruby issue](https://bugs.ruby-lang.org/issues/12724)) why it is done that way in a comment. Otherwise someone might come along later, wonder why it is done this way, and change it back.